### PR TITLE
changed!!! Add my project

### PR DESCRIPTION
**Pull request by VadimBezhkov**
Corrected notes

### CODE REVIEW

File: Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/ManagerProgramm.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            image.EventResultInfo += WorkWithText.ResultInfo;
+            image.EventResultRenameInfo += WorkWithText.ResultRenameInfo;
+
+            WorkWithText.Info();
+
+            while (true)
+            {
+                WorkWithText.ActionMenu();
+                CancellationTokenSource cancelTokenSource = new CancellationTokenSource();
+                CancellationToken token = cancelTokenSource.Token;
```
Comment (Audyman) [inline Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/ManagerProgramm.cs:25]: Better to move into very begin of the programm - Program => main method

File: Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/WorkWithText.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+namespace ConsoleApp14
+{
+    class WorkWithText
+    {
+        public static void Info()
+        {
+            Console.WriteLine(new string('-', 119));
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("\t \t \t \t \t Homework Bulk Thumbnail  Creator");
```
Comment (Audyman) [inline Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/WorkWithText.cs:15]: static text can be moved to a separate class Constants

**Review by Audyman — APPROVED** [review]
Please remove changes for database task from this commit - it should be in a dedicated PR.

**Comment by VadimBezhkov** [comment]
Thank you

**Pull request merged**